ci/cached-builds/gha_pr_changed_files.py

```diff
@@ -106,5 +114,9 @@ def test_get_build_directory(self):
         directory = get_build_directory("rocm-jupyter-pytorch-ubi9-python-3.11")
         assert directory == "jupyter/rocm/pytorch/ubi9-python-3.11"
 
+    def test_get_build_dockerfile(self):
+        directory = get_build_dockerfile("rocm-jupyter-pytorch-ubi9-python-3.11")
+        assert directory == "jupyter/rocm/pytorch/ubi9-python-3.11/Dockerfile.rocm"
```

Comment:
Should probably rather be:
```suggestion
        dockerfile = get_build_dockerfile("rocm-jupyter-pytorch-ubi9-python-3.11")
        assert dockerfile == "jupyter/rocm/pytorch/ubi9-python-3.11/Dockerfile.rocm"
```
but it's not important, just for readability a bit.